projects/Mallard/metro.config.js

```diff
@@ -6,12 +8,13 @@
  */
 
 module.exports = {
-  transformer: {
-    getTransformOptions: async () => ({
-      transform: {
-        experimentalImportSupport: false,
-        inlineRequires: false,
-      },
-    }),
-  },
-};
+    transformer: {
```

Comment:
there's a couple of lines in this pr that seem to be the effect of using the wrong auto code formatting options :(

don't think any of them are breaking but it's not ideal because the next time other dev touches any line in these files everything in them will change to their formatting

Reply:
I'm getting some strangeness from my auto formatter install in vscode too :(

Reply:
Is it ESLint?

Reply:
As put in the chat, I actually think these changes are coming from our `.editorconfig` file